keys.go

```diff
@@ -192,6 +206,40 @@ func (c *Client) GetKey(ctx context.Context, id string) (*Key, error) {
 	return &keys.Keys[0], nil
 }
 
+//GetKeyMetadata retrieves key metadata details by ID.
```

Comment:
```suggestion
// GetKeyMetadata retrieves and returns only the metadata information of a Key. 
// This can be useful if you don't have full permissions on a key, such as access to the payload data of an exportable key.
```